source/cfp.rst

```diff
@@ -0,0 +1,63 @@
+Call For Proposals
+==================
+
+We are planning to use `papercall.io`_ for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+Talk details
+------------
+
+Title
+  No help text
+Abstract
+  The abstract is the 300 character elevator pitch for this talk.
+Talk Format
+  What format is this talk best suited for?
+  Options: Talk; Workshop or tutorial; Code clinic, lunchtime event or other themed gathering; Poster
+Audience Level
+  Who is the best target audience for this talk?
+  Options: All; Beginner; Intermediate; Advanced
```

Comment:
Can we add something along the lines of:

Who will your session be most relevant to?

Options: General programmers, Web developers, Researchers (including scientists), Educators, Students, ...